tests/observability/metrics/utils.py

```
@@ -1342,3 +1354,86 @@ def get_vmi_guest_os_kernel_release_info_metric_from_vm(
+            bitmath.parse_string_unsafe(
+                PersistentVolumeClaim(
+                    name=vm_dv_templates[0].metadata.name,
+                    namespace=vm.namespace,
+                ).instance.spec.resources.requests.storage
+            ).Byte.bytes
+        )
+    )
+
+
+def get_vm_virt_launcher_pod_requested_memory(vm: VirtualMachineForTests) -> int:
+    if containers: = vm.vmi.virt_launcher_pod.instance.spec.containers:
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)